fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Lao.java

```diff
@@ -181,34 +184,57 @@ public void updateTransactionHashMap(List<PublicKey> attendees) {
     transactionHistoryByUser = new HashMap<>();
   }
 
-  // add a function that update all the transaction
+  /**
+   * Function that update all the transaction Update transactionByUser (current state of money)
+   * Update transactionHistory (current transaction perform per user)
+   *
+   * @param transactionObject object which was posted and now should update the lao map
+   */
   public void updateTransactionMaps(TransactionObject transactionObject) {
     if (transactionObject == null) {
       throw new IllegalArgumentException("The transaction is null");
     }
-    // Change the transaction per public key in transacionperUser
-    // for the sender and the receiver
+    /* Change the transaction per public key in transacionperUser
+    for the sender and the receiver*/
     if (this.getRollCalls().values().isEmpty()) {
       throw new IllegalStateException("A transaction need a roll call creation ");
     }
     if (this.pubKeyByHash.isEmpty()) {
       throw new IllegalStateException("A transaction need attendees !");
     }
 
-    // Contained in the receiver there are also the sender
-    // which has to be in the list of attendees of the roll call
+    /* Contained in the receiver there are also the sender
+    which has to be in the list of attendees of the roll call*/
     Iterator<PublicKey> receiversIte =
         transactionObject.getReceiversTransaction(pubKeyByHash).iterator();
     while (receiversIte.hasNext()) {
       PublicKey current = receiversIte.next();
```

Comment:
Don't hesitate to use foreach declarations :

```suggestion
    for (PublicKey current : transactionObject.getReceiversTransaction(pubKeyByHash)) {
```